src/components/chart/example-stacked-column-chart.njk

```diff
@@ -60,6 +60,8 @@
         },
         "yAxis": {
             "title": "Percentage of GDP (%)"
-        }
+        },
+        "percentageHeightDesktop": 50,
+        "percentageHeightMobile": 120
```

Comment:
120 feels very big for the examples. I think it would be good to reduce this number for the examples

Reply:
I've adjusted this now if you could check again.

Reply:
I've reduced it a little - but also added a max-width to the chart component as it won't render very wide on the page.